domains/user0.json

```diff
@@ -0,0 +1,13 @@
+{ 
+           "description": "My own web profile",
+           "repo": "https://github.com/user0-07161/user0-07161.github.io",
+           "owner": {
+           "username": "user0-07161",
+           "email": "user0thenyancat@proton.me",
+           "discord": "1213799919920484364"
+        },
+
+        "record": {
+            "CNAME": "user0-07161.github.io"
```

Comment:
You need to have content present on your site.

Reply:
@DEV-DIBSTER
Should be fixed now.